strider/fetcher.py

```diff
@@ -253,6 +258,8 @@ def get_kp_request_body(
         curie: str,
         step: Step,
         kp: dict,
+
+
```

Comment:
What's the deal with these extra newlines?

Reply:
Not sure, but I removed them.